hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MultiVersionConcurrencyControl.java

```diff
@@ -272,33 +252,59 @@ public long getWritePoint() {
    */
   @InterfaceAudience.Private
   public static class WriteEntry {
-    private final long writeNumber;
+    private long writeNumber = 0L;
```

Comment:
Because it only written by a single, thread this is safe to do.... it doesn't need to be volatile.